packages/mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -10,6 +10,8 @@
   <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
   <uses-permission android:name="android.permission.CAMERA" />
 
+  <uses-sdk android:minSdkVersion="16" />
```

Comment:
Adding this back, I mistakenly removed the line in my last PR but should have just removed the targetSDK field in it